public/tinymce/skins/woocommerce/editor.css

```diff
@@ -0,0 +1,8 @@
+.mce-container.mce-menu,
+.mce-container .mce-floatpanel.mce-popover {
+	z-index: 25 !important;
```

Comment:
Just a note for history sake: This is overwriting another `!important` rule that gets set elsewhere. Maybe in the future we can untangle some of the post specific stuff from TinyMCE so we don't need this.